src/main/java/seedu/duke/assets/Medicine.java

```diff
@@ -1,11 +1,20 @@
 package seedu.duke.assets;
 
 public class Medicine {
-    String medicineId;
-    String medicineName;
-    String dosage;
-    String expiry;
-    String sideEffects;
-    int quantity;
+    private int medicineId;
+    private String medicineName;
+    private int dosage; //in milligrams
```

Comment:
int or double better? Can we specify the variable name from dosage to milligramDosage?